src/sass/_overlay_panel.scss

```diff
@@ -0,0 +1,25 @@
+.overlay-panel {
+  // mobile
+  height: 100vh;
+  padding-bottom: 0.75rem;
+  padding-top: 0.5rem;
+  position: absolute;
+  width: 100vw;
+  z-index: 103 !important;
+
+  @include medium {
+    width: 500px;
+  }
+
+  @include large {
+    width: 650px;
+  }
```

Comment:
I think we should use the standard from vanilla for `is-wide` and `is-narrow` and avoid specifying our own sizes.

Reply:
this might require a bit of experiment as `is-narrow` is a little too narrow for the contents on some of the permissions side panels but `is-wide` is too wide :sweat_smile: In any case, I'd leave this for now as `_overlay_panel.scss` is currently just a place holder and will most likely be adjusted in the permissions PR.